### DISCUSSION TITLE
some pits could not be found by basin detection

### DISCUSSION THREAD

**Discussion by HusBunia** [brainvisa/brainvisa.github.io#141] [Q&A] [answered]
Hi Dear Brainvisa expert,
I have processed a series brain surface and generate the sulcal pits atlas 
However, I found some small basins could not be defined as a complete atlas parcel. Thus my group average atlas could not also found these small basins.
To illustrate this, please refer to the basins highlighted in red circles in a random subject's brain scans linked below (the third image shows the dpf distribution):
<img width="324" alt="1" src="https://github.com/brainvisa/brainvisa.github.io/assets/40214939/768ae213-dc73-4606-ad6d-ea6aeb308b02">
<img width="308" alt="2" src="https://github.com/brainvisa/brainvisa.github.io/assets/40214939/a88ea700-5071-4024-afb5-ec45015ea660">
<img width="394" alt="3" src="https://github.com/brainvisa/brainvisa.github.io/assets/40214939/96a42585-a75e-4703-ac22-830c8c986f78">
This result seems inconsistent with the atlas representation in published articles, especially since my focus is primarily on the basins in the frontal cortex. Is this outcome correct given the methodology applied? the entire pipeline was adapted from (https://github.com/YannLeGuen/sulcal_pits_analysis). The fiedler length and area calculations were computed from my own dataset and their results are very similar to the default values.
To better understand and validate my results, I would like to ask if there's any possibility to access the atlas you've created from the OASIS dataset. Having this comparative data would significantly aid my analysis.
Really appreciated for any help ! 
Thanks!

**Comment by gauzias** [comment, marked as answer]
Hi,
the DPF and sulcal basins from your figures seem fine to me, and it is expected that some pits/basins observed at the individual level do not match the atlas, so I do not really get your point.
As described in our papers (https://www.sciencedirect.com/science/article/abs/pii/S1053811915001007?via%3Dihub, as well as in Leguen et al., Cereb Cortex 2017 and others) a one-to-one correspondence between individual sulcal pits and any atlas is not expected.
Another perspective on the matching between individual basins and an atlas can be found in the following paper:
https://www.sciencedirect.com/science/article/abs/pii/S1361841520301134?via%3Dihub
I can share the atlas we computed from 138 individuals from the OASIS database in Auzias et al., Neuroimage 2015 but since we did not use freesurfer for interpolation across individuals, it will not be possible for you to map your individual data on it.
But more importantly, even with our atlas, you will observe the same mismacth for some individuals.

**Reply by HusBunia** [reply]
Hi,
Apologies for my late response.
If the result is good, I believe I should no worry about the atlas.  Thanks for your help ! 
I tried to detecting sulcal pits within some classical brain regions, like the superior frontal gyrus from the DK atlas. Upon generating a pits-atlas, I noticed three basins in this region. Interestingly, in the pits-atlas (which I've circled in blue in the attached images), two of the basins are notably significant, while the others primarily consist of one large basin. I also examined some individual basin files and observed that what appears to be two separate basins in some regions have been treated as one.
Now, I understand that the group atlas is linked to pits density and that there isn’t a strict one-to-one correspondence between individual and template. My previous concern is the interpretation of the biological significance of these three basins. I want to ensure that my understanding and analysis are aligned with the underlying biology.
If there are any misconceptions in my approach, I would appreciate your guidance. Thank you once again for your help and support!
